commands/inits/data/aliases.fish

```diff
@@ -0,0 +1,6 @@
+alias gs='scmpuff_status'
+alias ga='git add'
+alias gd='git diff'
+alias gl='git log'
+alias gco='git checkout'
+alias grs='git reset'
```

Comment:
Since this file is totally identical to `aliases.sh`, I believe we should not fork for fish (unless I'm missing something), as it creates more than one place the defaults would be adjusted, and creates the possibility of drift.